src/graphics/gfx_print.cpp

```diff
@@ -235,26 +300,63 @@ void SuperPrint(int SuperN, const char* SuperChars, int Font, float X, float Y,
 
 int SuperTextPixLen(const char* SuperChars, int Font)
 {
-    return SuperTextPixLen(SDL_strlen(SuperChars), SuperChars, Font);
+    int dFont = FontManager::fontIdFromSmbxFont(Font);
```

Comment:
For all of these, maybe better to check render mode before looking for font?

Reply:
Do you mean to make a kind of:
```cpp
int dFont = NewFontRender ? FontManager::fontIdFromSmbxFont(Font) : -1;
```
?

Reply:
Done.